server/api/tumdev/campus_backend.proto

```diff
@@ -126,12 +126,11 @@ service Campus {
     option (google.api.http) = {get: "/movies/{last_id}"};
   }
 
-  rpc SendFeedback(SendFeedbackRequest) returns (SendFeedbackReply) {
-    option (google.api.http) = {post: "/feedback"};
-  }
-
-  rpc SendFeedbackImage(SendFeedbackImageRequest) returns (SendFeedbackImageReply) {
-    option (google.api.http) = {post: "/feedback/{id}/{image_nr}"};
+  rpc SendFeedback(stream SendFeedbackRequest) returns (SendFeedbackReply) {
```

Comment:
`SaveFeedback` might be a better name?

Reply:
Given the naming of the other APIs, `NewFeedback` is likely a better name as this is more self-consistent

Reply:
The standard for good APIs suggests Create in that case: https://google.aip.dev/133

Reply:
That is a good resource ⇒ have changed across all other apis (except `SearchRooms` and the weird ios one which is 🪄) the naming accordingly